### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by nokite**
### PR's key points
- Updated some dependencies (gradle, kotlin, libraries etc)
- Updated Project.xml (automatically done by Android Studio)
- Updated from the deprecated classifier to archiveClassifier

### How to review this PR?
- Check the latest versions of the libs
- Open the project on master with Android Studio and see Project.xml
- https://docs.gradle.org/current/dsl/org.gradle.api.tasks.bundling.Jar.html#org.gradle.api.tasks.bundling.Jar:classifier

### Definition of Done
- [ ] Tests added (if new code is added) - Does not apply
- [x] There is no outcommented or debug code left

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```
Comment (adriangl) [inline gradle/wrapper/gradle-wrapper.properties:3]: I suggest bumping the version to `6.9`. For now I'd like to postpone upgrading to version 7.x until we can ensure compatibility with newer Android Gradle Plugin versions

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
```

**Review by adriangl — APPROVED** [review]
Thanks for the contribution @nokite! I guess the plugin works as expected with the version bumps, right?

If you can, please add the changes to CHANGELOG.md!

**Comment by nokite** [comment]
> Thanks for the contribution @nokite! I guess the plugin works as expected with the version bumps, right?
> 
> If you can, please add the changes to CHANGELOG.md!

Yes, I tested it on my project and the behavior is the same. No issues with your change to gradle 6.9 either 👍
Sure thing, I'm adding everything to the CHANGELOG now.

I should mention that there are some warnings when executing gradle tasks. They were also shown before the dependencies were updated, though there's a new line now:

`The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.4.20` that might work differently than in the requested version `1.5.20`.`
It seems like we can ignore this for now though.
https://youtrack.jetbrains.com/issue/KT-41142

This one is also new, and also seems safe for now:
`w: Language version 1.3 is deprecated and its support will be removed in a future version of Kotlin`

**Comment by nokite** [comment]
All warnings:

```
> Configure project :
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.4.20` that might work differently than in the requested version `1.5.20`.

> Task :compileKotlin
The `kotlin-dsl` plugin applied to root project 'poeditor' enables experimental Kotlin compiler features. For more information see https://docs.gradle.org/6.9/userguide/kotlin_dsl.html#sec:kotlin-dsl_plugin
w: Language version 1.3 is deprecated and its support will be removed in a future version of Kotlin
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /Users/.../.gradle/wrapper/dists/gradle-6.9-all/dooywd8nv05k16orzxge2b1bs/gradle-6.9/lib/kotlin-stdlib-1.4.20.jar (version 1.4)
    /Users/.../.gradle/wrapper/dists/gradle-6.9-all/dooywd8nv05k16orzxge2b1bs/gradle-6.9/lib/kotlin-stdlib-common-1.4.20.jar (version 1.4)
    /Users/.../.gradle/wrapper/dists/gradle-6.9-all/dooywd8nv05k16orzxge2b1bs/gradle-6.9/lib/kotlin-stdlib-jdk7-1.4.20.jar (version 1.4)
    /Users/.../.gradle/wrapper/dists/gradle-6.9-all/dooywd8nv05k16orzxge2b1bs/gradle-6.9/lib/kotlin-stdlib-jdk8-1.4.20.jar (version 1.4)
    /Users/.../.gradle/wrapper/dists/gradle-6.9-all/dooywd8nv05k16orzxge2b1bs/gradle-6.9/lib/kotlin-reflect-1.4.20.jar (version 1.4)
    /Users/.../.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.4.31/e613be5465ef1e6fd0468707690b7ebf625ea2fe/kotlin-stdlib-jdk8-1.4.31.jar (version 1.4)
    /Users/.../.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.4.31/63db9d66c3d20f7b8f66196e7ba86969daae8b8a/kotlin-reflect-1.4.31.jar (version 1.4)
    /Users/.../.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.4.31/84ce8e85f6e84270b2b501d44e9f0ba6ff64fa71/kotlin-stdlib-jdk7-1.4.31.jar (version 1.4)
    /Users/.../.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.5.20/9de35cc611bcecec8edce1d56d8e659953806751/kotlin-stdlib-1.5.20.jar (version 1.5)
    /Users/.../.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.5.20/3d79dbd48bf605f4aac1e7028981a1953e245cbb/kotlin-stdlib-common-1.5.20.jar (version 1.5)
w: Consider providing an explicit dependency on kotlin-reflect 1.5 to prevent strange errors
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
w: /Users/.../src/main/kotlin/com/hyperdevs/poeditor/gradle/ktx/DocumentExtensions.kt: (25, 8): This class shouldn't be used in Kotlin. Use kotlin.Boolean instead.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.

```

**Comment by adriangl** [comment]
@nokite Could you resolve the conflicts so I can merge the PR?

**Pull request merged**